va/va.h

```diff
@@ -3076,6 +3076,10 @@ typedef struct _VAPictureParameterBufferH264
         uint32_t value;
     } pic_fields;
     uint16_t frame_num;
+    /** \brief Same as the H.264 bitstream syntax element. */
+    uint8_t num_ref_idx_l0_default_active_minus1;
+    /** \brief Same as the H.264 bitstream syntax element. */
+    uint8_t num_ref_idx_l1_default_active_minus1;
 
     /** \brief Reserved bytes for future use, must be zero */
     uint32_t                va_reserved[VA_PADDING_MEDIUM];
```

Comment:
I understand that you add these field between the crevice of alignment, have you checked the size both on 64bit and 32 bit?

Reply:
Yes, there is no difference between 64-bit (x86_64) and 32-bit (arm-v7a): In both cases, `sizeof(VAPictureParameterBufferH264)` is 672 bytes before and after the change. All member offsets stay the same.

`sizeof(VASliceParameterBufferH264)` is 3128 bytes before and after the change on both 64-bit and 32-bit. The only member offset that changes is `va_reserved`, which moves from 3112 to 3120.

Reply:
Shouldn't va_reseverd size be reduced to maintain overall structure size ?

Reply:
Overall structure size is maintained by placing the new elements inside previously unused 32-bit alignment padding, see the comment above.